Sources_v3/StreamChatUI/ChatChannel/ChatChannelMessageComposerView.swift

```diff
@@ -21,152 +24,156 @@ open class ChatChannelMessageComposerView<ExtraData: UIExtraDataTypes>: UIInputV
     
     public private(set) lazy var messageInputView: ChatChannelMessageInputView<ExtraData> = .init(uiConfig: uiConfig)
     
-    public private(set) lazy var sendButton: UIButton = {
-        let button = UIButton().withoutAutoresizingMaskConstraints
-        if #available(iOS 13.0, *) {
-            button.setImage(UIImage(systemName: "arrow.up.circle.fill"), for: .normal)
-        }
-        button.widthAnchor.constraint(equalTo: button.heightAnchor, multiplier: 1).isActive = true
-        return button
-    }()
-        
-    public private(set) lazy var searchButton: UIButton = {
-        let button = UIButton().withoutAutoresizingMaskConstraints
-        if #available(iOS 13.0, *) {
-            button.setImage(UIImage(systemName: "magnifyingglass"), for: .normal)
-        }
-        button.widthAnchor.constraint(equalTo: button.heightAnchor, multiplier: 1).isActive = true
-        return button
+    public private(set) lazy var sendButton: MessageComposerSendButton = {
+        MessageComposerSendButton().withoutAutoresizingMaskConstraints
     }()
     
     public private(set) lazy var attachmentButton: UIButton = {
-        let button = UIButton().withoutAutoresizingMaskConstraints
-        if #available(iOS 13.0, *) {
-            button.setImage(UIImage(systemName: "paperclip"), for: .normal)
-        }
-        button.widthAnchor.constraint(equalTo: button.heightAnchor, multiplier: 1).isActive = true
-        return button
+        ChatSquareButton(
+            image: UIImage(named: "clip", in: .streamChatUI)!,
+            tintColor: .squareButtonTint
+        ).withoutAutoresizingMaskConstraints
     }()
-
-    public private(set) lazy var boltButton: UIButton = {
-        let button = UIButton().withoutAutoresizingMaskConstraints
-        if #available(iOS 13.0, *) {
-            button.setImage(UIImage(systemName: "bolt.fill"), for: .normal)
-        }
-        button.widthAnchor.constraint(equalTo: button.heightAnchor, multiplier: 1).isActive = true
-        return button
+    
+    public private(set) lazy var commandsButton: UIButton = {
+        ChatSquareButton(
+            image: UIImage(named: "bolt", in: .streamChatUI)!,
+            tintColor: .squareButtonTint
```

Comment:
Isn't this against the customization? I mean, shouldn't the color be setup in `defaultAppearance`?

Reply:
Tint stuff updated, check 4336e5f